src/components/pageNav/index.jsx

```diff
@@ -4,30 +4,30 @@ import { Link } from 'gatsby';
 import Button from '../site/button';
 import ArrowLeft from '../site/icons/ArrowLeft';
 import ArrowRight from '../site/icons/ArrowRight';
+import styles from './pageNav.module.scss';
 
 class PageNav extends React.PureComponent {
   renderNavItem(type, item) {
     return (
       <Button
-        flat
-        primary
-        swapTheming
+        icon
+        className="outlined"
         to={item.link === '' ? '/' : item.link}
         component={Link}
         iconEl={type === 'previous' ? <ArrowLeft /> : <ArrowRight />}
         iconBefore={type === 'previous'}
-      >
-        {type === 'previous' ? 'Back' : 'Continue'}
-      </Button>
+        tooltipLabel={type === 'previous' ? 'Back' : 'Continue'}
```

Comment:
Please update the tooltip to use the previous and next section titles as the tooltip text.  Sorry I didn't make this clear in the description of [EPO-2320](https://jira.lsstcorp.org/browse/EPO-2320).  Checkout the pdf design file attached to that ticket to see what I mean.  That text should already be available on the previous and next props coming into the PageNav component. Those objects look like this:

```
next {
   title
   link
}
previous {
   title
   link
}
```

Reply:
I updated the code to reflect the changes and push it up into the branch.